vocab/security/vocabulary.yml

```
@@ -39,9 +39,18 @@ class:
       - label: vc-data-integrity
         url: https://www.w3.org/TR/vc-data-integrity/#dataintegrityproof
 
+  - id: Multikey
+    label: Multikey Verification Method
+    upper_value: sec:VerificationMethod
+    comment: Verification method to be used with, for example, data integrity proof cryptographic suites, such as the eddsa-2022 cryptographic suite. See the <a href="https://www.w3.org/TR/vc-di-eddsa/#multikey">EdDSA Cryptosuite v2022 specification</a> for further details.
+    see_also:
+      - label: eddsa-2022
```

```suggestion
      - label: EdDSA Cryptosuite v2022
```

I don't know why we're labeling this with eddsa-2022, since Multikey is a key format, not a specific cryptosuite? Maybe the following above?